bitrise.yml

```diff
@@ -21,49 +24,16 @@ workflows:
         title: Step Test
         inputs:
         - packages: github.com/bitrise-steplib/steps-go-test
-
-  go-tests:
-    before_run:
-    steps:
-    - script:
-        title: Export go files to test
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            no_vendor_paths="$(go list ./... | grep -v vendor)"
-            envman add --key GOLIST_WITHOUT_VENDOR --value "$no_vendor_paths"
-    - script:
-        title: Err check
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            go get -u -v github.com/kisielk/errcheck
-            errcheck -asserts=true -blank=true $GOLIST_WITHOUT_VENDOR
-    - script:
-        title: Go lint
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            go get -u -v github.com/golang/lint/golint
-            while read -r line; do
-              echo "-> Linting: $line"
-              golint_out="$(golint $line)"
-              if [[ "${golint_out}" != "" ]] ; then
-                echo "=> Golint issues found:"
-                echo "${golint_out}"
-                exit 1
-              fi
-            done <<< "$GOLIST_WITHOUT_VENDOR"
     - script:
-        title: Go test
+        title: Output test
         inputs:
         - content: |-
-            #!/bin/bash
-            set -ex
-            go test $(go list ./... | grep -v vendor)
+            #!/usr/bin/env bash
+            echo "code coverage report: $BITRISE_DEPLOY_DIR/coverage.txt"
+            if [[ -z "$BITRISE_DEPLOY_DIR/coverage.txt" ]]; then
+              echo "error: there is no $BITRISE_DEPLOY_DIR/coverage.txt env var specified"
```

Comment:
please write a more understandable error message here

Reply:
please use more descriptive name than `coverage.txt`, all the tools are exporting with the same name. (in case you change the step to have output env(s)